api/src/main/java/com/github/retrooper/packetevents/protocol/world/Dimension.java

```diff
@@ -23,7 +23,7 @@
 
 public class Dimension {
     private int id;
-    private NBTCompound attributes;
+    private NBTCompound attributes = new NBTCompound();
```

Comment:
Never do it like this, use the constructor.

Reply:
Why may I ask?